src/components/misc/personViews/PersonViewTable.jsx

```diff
@@ -132,7 +132,24 @@ export default class PersonViewTable extends React.Component {
                             const val0 = row0.data.content[this.state.sortIndex] || '';
                             const val1 = row1.data.content[this.state.sortIndex] || '';
 
-                            let x = val0.toString().localeCompare(val1.toString());
+                            let x = 0;
+
+                            if (Array.isArray(val0) && typeof(val0[0]) != 'undefined' && typeof(val1[0]) != 'undefined') {
+                                x = val0[0].text.toLowerCase().toString().localeCompare(val1[0].text.toLowerCase().toString());
+                            }
+                            else {
+                                x = val0.toString().localeCompare(val1.toString());
+                            }
+
+                            if(typeof(val0[0]) == 'undefined' && typeof(val1[0]) != 'undefined') {
+                                return 1;
+                            }
+                            if(typeof(val0[0]) != 'undefined' && typeof(val1[0]) == 'undefined') {
+                                return -1;
+                            }
+                            if(typeof(val0[0]) == 'undefined' && typeof(val1[0]) == 'undefined') {
+                                return 0;
+                            }
```

Comment:
These should only happen if we are working with arrays. They could be enclosed in an if statement checking that `Array.isArray(val0) && Array.isArray(val1)`.

This is already what you're doing on line 137, so it would make sense to put all of your new code in the same if statement verifying that we are indeed working with arrays here.